src/main/java/ai/deepcode/javaclient/core/LoginUtilsBase.java

```diff
@@ -9,6 +9,7 @@
 import java.util.HashSet;
 import java.util.Set;
 
+@SuppressWarnings("unused")
```

Comment:
Is this class used somewhere outside to have it here?

Reply:
Potentially yes, as it's having public methods to be used by sdk users.

Reply:
It's just IntelliJ that uses it to my knowledge (correct me @ArtsiomCh if wrong). Let's clean it up if there are no sdk users. How come Login is part of SDK anyway?